packages/react/index.js

```diff
@@ -76,7 +76,5 @@ export {
   DEPRECATED_createResponder,
   unstable_createFundamental,
   unstable_createScope,
-  jsx,
-  jsxs,
-  jsxDEV,
 } from './src/React';
+export {jsx, jsxs, jsxDEV} from './src/jsx/ReactJSX';
```

Comment:
Same, this puts them on React object again for the npm build. I don't think we want them here?